tests/unit/crypto_tests.py

```diff
@@ -34,6 +34,17 @@ def test_roundtrip(self):
         self.signer.validate_signature(self.message, signature)
 
     @mock.patch("time.time")
+    def test_signature_info(self, time):
+        time.return_value = 0
+        signature = self.signer.make_signature(self.message, max_age=datetime.timedelta(seconds=30))
+
+        time.return_value = 10
```

Comment:
Why is this line necessary? The expiration won't change as the time changes, so I'm just curious